client/rest-high-level/src/test/java/org/elasticsearch/client/documentation/CRUDDocumentationIT.java

```diff
@@ -932,6 +932,49 @@ public void onFailure(Exception e) {
         }
     }
 
+    public void testExists() throws Exception {
+        RestHighLevelClient client = highLevelClient();
+        // tag::exists-request
+        GetRequest getRequest = new GetRequest(
+            "posts", // <1>
+            "doc",   // <2>
```

Comment:
would "_doc" be better here ?

Reply:
I don't think it makes a difference to be honest.

Reply:
Oh, it does not ;)
I was thinking of #27816 and #27751 ( Docs should use `_doc` as a type name whenever possible ). I do not think that it is consistently done in the docs for the high level client though. Just a thought ...